CHANGES.md

```diff
@@ -36,6 +36,7 @@
 - #1796: Don't show "back" arrow navigation (on mobile) in the chat header when in `singleton` mode
 - #1821: Errors caused by malformed URLs are now handled
 - #1819: Click on a desktop notification now opens corresponding chat.
+- #1820: Set focus on jid field after controlbox is loaded
```

Comment:
Please rebase and move this line to the top of the changelog.

Otherwise, looks good.

Reply:
Done